animation-timing/callback-exception.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+  <head>
+    <title>requestAnimationFrame callback exception reported to window.onerror</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://w3c.github.io/web-performance/specs/RequestAnimationFrame/Overview.html#dom-windowanimationtiming-requestanimationframe"/>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+      var custom_exception = 'requestAnimationFrameException';
+      async_test(function (t) {
+        setup({allow_uncaught_exception : true});
+        window.onerror = function(e) {
+          t.step(function() {
+             assert_equals(e,'Error: '+custom_exception);
```

Comment:
right, the fn returns first argument "Uncaught Error : <message>" in Chrome, while "Error : <message>" in FF. so i will change it, but no argument concretely provides the error message only!